README.md

```diff
@@ -56,7 +56,9 @@ lydia -l ltlf -f examples/counter_1.ltlf --part examples/counter_1.part
 ```
```

Comment:
check the README

Reply:
what about adding acknowledgements to Whitemech and Tailor?

Reply:
right, will do :-)